CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+* Added multiArgs argument for promisifyAll when calling Docker.run
```

Comment:
Can you talk about what is fixed/the user impact, rather than what was changed (ie, think of it as a user reading the changelog, you want to know how it impacts your usage)